cf-check/validate.c

```diff
@@ -383,11 +391,6 @@ static void UpdateValidatorLock(
 static bool ValidateMDBValue(
     ValidatorState *state, MDB_val value, const char *name)
 {
-    if (value.mv_size <= 0)
-    {
-        ValidationError(state, "0 size %s", name);
-        return false;
-    }
     if (value.mv_data == NULL)
```

Comment:
Hmmm, interesting. I would expect this to be the case as well when `mv_size == 0`.